### ISSUE TITLE
Export PDF: The last line words sometimes cut into 2 parts.

### ISSUE THREAD

**Issue by tangshi** [uranusjr/macdown#298]
**As shown below:**    

![2015-02-25 22 44 21](https://cloud.githubusercontent.com/assets/5255026/6372569/271dd6ba-bd40-11e4-83af-ef7d8b4f6174.png)

**Comment by thomaskilian** [comment]
This also happens with latin characters. A fix here is appreciated.

**Comment by Jmuccigr** [comment]
Print to PDF is fine though?

**Comment by thomaskilian** [comment]
No. I have 2 files where lines are split like in the above example. Do you need the source?
